### ISSUE TITLE
Early-stage bike shedding

### ISSUE THREAD

**Issue by stevekinney** [turingschool/monocle#1]
@jcasimir, I'm kind of thinking this could be an API-first project. Not to bike shed too early, but I'm thinking that the front-end would be easier to build if it wasn't in the asset pipeline. Second thought is that it would be super cool to use this as an opportunity to teach how to both produce and consume a [JSON API](http://jsonapi.org/)-compliant API. I think Rails 5 _might_ support this by default, but I'm a little out of touch with Rails these days.

**Comment by jcasimir** [comment]
yeah I'm down with thinking about the API from the start / doing that first.

**Comment by calaway** [comment]
@stevekinney, I'm going to be working on this for two weeks starting today as my Mod 3 final project. 

After looking into it a bit I don't think Rails 5 supports the JSON API standard right out of the box, but [this link](http://jsonapi.org/implementations/#server-libraries-ruby) shows several resources that could help. Two that stand out to me are the [ActiveModel::Serializers](https://github.com/rails-api/active_model_serializers), which I think should be fairly lightweight and simple, or [JSONAPI::Utils](https://github.com/b2beauty/jsonapi-utils), which looks more full featured and built with Rails in mind.

Do you think anyone has a specific preference on which is used, or is anything fine as long as it works?
